.devcontainer/Dockerfile

```
@@ -0,0 +1,23 @@
+
+RUN mkdir -p /usr/local/share/npm-global && chown -R node:node /usr/local/share
+ARG USERNAME=node
+RUN SNIPPET="export PROMPT_COMMAND='history -a' && export HISTFILE=/commandhistory/.bash_history" && mkdir /commandhistory && touch /commandhistory/.bash_history && chown -R $USERNAME /commandhistory
+ENV DEVCONTAINER=true
+RUN mkdir -p /workspace /home/node/.claude /home/node/.claude-backup && chown -R node:node /workspace /home/node/.claude /home/node/.claude-backup
+WORKDIR /workspace
+USER node
+ENV NPM_CONFIG_PREFIX=/usr/local/share/npm-global
+ENV PATH=$PATH:/usr/local/share/npm-global/bin
+ENV SHELL=/bin/zsh
+RUN sh -c "$(wget -O- https://github.com/deluan/zsh-in-docker/releases/download/v1.2.0/zsh-in-docker.sh)" -- -p git -p fzf -a "source /usr/share/doc/fzf/examples/key-bindings.zsh" -a "source /usr/share/doc/fzf/examples/completion.zsh" -a "export PROMPT_COMMAND='history -a' && export HISTFILE=/commandhistory/.bash_history" -x
```

Hadolint (2.12.0)

[info] Avoid use of wget without progress bar. Use `wget --progress=dot:giga <url>`. Or consider using `-q` or `-nv` (shorthands for `--quiet` or `--no-verbose`).

(DL3047)